### DISCUSSION TITLE
Lightweight indicators

### DISCUSSION THREAD

**Discussion by RSalgaonkar** [tradingview/lightweight-charts#933] [General] [answered]
Hi, can we add basic indicators like exponential moving average, Bollinger bands and volume weighted average price to charts?

**Comment by timocov** [comment, marked as answer]
Hi there,

we don't have any plans to add any calculations to the library because we don't to keep it as smaller as possible with only needed charting features. Probably we'll add a separate package with indicators, but as I said we don't have any plans for that now.

But you can easily implement this yourself, see https://github.com/tradingview/lightweight-charts/discussions/647

**Reply by RSalgaonkar** [reply]
Thank you for quick feedback. will look into implementation.
